.pylintrc

```diff
@@ -12,4 +12,10 @@ disable=
     unnecessary-pass,
     wrong-import-position,
     logging-format-interpolation,
-    too-many-lines
```

Comment:
I think we should remove these. Instead of this, we can change pre-commit file to contain:
```
#!/usr/bin/env bash
git-pylint-commit-hook --ignore=features/ --ignore=fakes --ignore=tests/
```

Reply:
Why ignore the whole folder?

Reply:
since features tests and fakes are used only for testing. We don't care about linting errors in them